ifrs17-template/Report/ParameterReports.ipynb

```diff
@@ -0,0 +1,521 @@
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var args = new ImportArgs(\"CH\", 2021, 3, default, default , default);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Data Node"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var dataNodeData = (await Workspace.LoadDataNodesAsync(args))",
+                "\n    .Values",
+                "\n    .ToArray();"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var dataNodeStates = (await Workspace.LoadCurrentAndPreviousParameterAsync<DataNodeState>(args, x => x.DataNode))",
+                "\n    .Values",
+                "\n    .SelectMany(x => x.Select(y => ",
+                "\n        new DataNodeStateReportParameter {",
+                "\n            GroupOfContract = y.Value.DataNode,",
+                "\n            Period = ((Period)y.Key),",
+                "\n            Year = y.Value.Year,",
+                "\n            Month = y.Value.Month,",
+                "\n            Scenario = y.Value.Scenario }",
+                "\n        ))",
+                "\n    .ToArray();",
+                "\n//dataNodeStates.Take(2)"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# YieldCurve"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var lockedYieldCurves = (await Workspace.LoadLockedInYieldCurveAsync(args, dataNodeData))",
+                "\n    //.Select(x => new { DataNode = x.Key, LockedIn = ( x.Value.Year, x.Value.Month, x.Value.Scenario, x.Value.Currency, x.Value.Name)})",
+                "\n    .Select(x => new YieldCurveReportParameter {",
+                "\n        GroupOfContract = x.Key,",
+                "\n        YieldCurveType = \"Locked-In Curve\",",
+                "\n        Year = x.Value.Year,",
+                "\n        Month = x.Value.Month,",
+                "\n        Scenario = x.Value.Scenario,",
+                "\n        Currency = x.Value.Currency,",
+                "\n        Name = x.Value.Name }",
+                "\n        )",
+                "\n    .ToArray();",
+                "\nvar currentYieldCurves = (await Workspace.LoadCurrentYieldCurveAsync(args, dataNodeData))",
```

Comment:
There's no property to distinguish between Current YC for period -1 and period 0. It will be noticeable only from Year Month if 2 different curves are taken, else also Year and Month are the same. 
Should we use the YieldCurveName to specify "for current period" - "for previous period"?